runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -67,9 +64,9 @@
 import net.runelite.client.util.Text;
 
 @PluginDescriptor(
-	name = "Slayer",
-	description = "Show additional slayer task related information",
-	tags = {"combat", "notifications", "overlay", "tasks"}
+		name = "Slayer",
+		description = "Show additional slayer task related information",
+		tags = {"combat", "notifications", "overlay", "tasks"}
```

Comment:
Although the IDE does 2 tabs automatically for annotations, we prefer one. Also avoid changing whitespacing unless the code was changed.